guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
-| `MONAI_HOSTNAME`    | `None`                                     | String              | Name of the host running the MAP.                |
-| `MONAI_HOSTVERSION` | `0.0.0`                                    | Semantic Version    | Version of the host running the MAP.             |
-| `MONAI_INPUTPATH`   | `/var/monai/input/`                        | Folder Path         | Path to the input folder for the Application.    |
-| `MONAI_JOBID`       | `00000000000000000000000000000000`         | `/[A-F0-9a-f]{32}/` | Unique identity of the job the MAP is executing. |
-| `MONAI_OUTPUTPATH`  | `/var/monai/output/`                       | Folder Path         | Path to the output folder for the Application.   |
-| `MONAI_TIMEOUT`     | `/etc/monai/app.json#timeout`              | Integer             | Timeout, in seconds, applied to the Application. |
-| `MONAI_WORKDIR`     | `/var/monai/`                              | Folder Path         | Path to the Application's working directory.     |
-| `MONAI_MODELPATH`   | `/opt/monai/models/`                       | Folder Path         | Path to the Application's models directory.     |
+    - Third-party licenses
 
+### Container Behavior and Interaction
 
-### Manifest Export
+A MAP is a single container supporting the following defined behaviors when started.
 
-The Executor is able to function in a special mode in which it will export the MAP's manifest files to a mounted folder.
-This enables external tooling and services to read the manifest without any special tooling beyond the ability to run the MAP correctly.
+#### Default Behavior
 
-The Executor MUST detect when the following folders have been mounted.
+When a MAP is started from the CLI or other means without any parameters, the MAP shall execute the contained application. The MAP internally may use `Entrypoint`, `CMD`, or a combination of both.
 
-- `/var/run/monai/export/app/`: when detected, the Executor will copy the contents of `/opt/monai/app/` to the folder.
+#### Manifest Export
 
-- `/var/run/monai/export/config/`: when detected, the Executor will copy `/etc/monai/app.json` and `/etc/monai/pkg.json` to the folder.
+A MAP SHALL provide at least one method to access the _embedded application_, _models_, _licenses_, _README_, or _manifest files_, namely, `app.json` and `package.json`.
 
-- `/var/run/monai/export/models/`: when detected, the Executor will copy the contents of `/opt/monai/models/` to the folder.
+- The Method SHALL have the option to print one or more manifest files to the console.
 
-- `/var/run/monai/export/`: when detected without any of the above being detected, the Executor SHALL:
+- The Method SHALL have the option to copy one or more manifest files to a mounted volume path.
 
-  - copy the contents of `/opt/monai/app/` to `/var/run/monai/export/app/`.
+  - `/var/run/holoscan/export/app/`: when detected, the Script copies the contents of `/opt/holoscan/app/` to the folder.
```

Comment:
Readers would be confused by the `Script`. They would not know how to build a package with the magic `Script` in the specification and wonder where it is from.

Reply:
Changed `Script` to `Method`